authors/apps/profiles/models.py

```diff
@@ -18,29 +19,49 @@ class Profile(TimeStampModel):
     country = models.CharField(_('country'), blank=True, null=True, max_length=100, default='')
     phone = models.IntegerField(_('phone'), blank=True, null=True, default=0)
     website = models.URLField(_('website'), blank=True, null=True, default='')
-    # follows = models.ManyToManyField('self', related_name='followed_by', symmetrical=False)
-    
+    follows = models.ManyToManyField('self', related_name='followed_by', symmetrical=False)
+    username = models.CharField(_('username'), max_length=100, blank=True, null=True)
```

Comment:
Could we refactor this to use a `SerializerMethodField` or `ReadOnlyField` in `ProfileSerializer` instead?

Reply:
Done